src/button/button.css

```diff
@@ -1,93 +1,204 @@
+input[type="button"].omrs-text-neutral {
+  color: rgba(19, 19, 21, 0.6);
+}
+
+button.omrs-text-neutral > i,
+input[type="submit"].omrs-text-neutral > i,
+input[type="button"].omrs-text-neutral > i {
+  color: var(--omrs-primary-blue);
+}
+
+button.omrs-text-neutral:hover,
+input[type="submit"].omrs-text-neutral:hover,
+input[type="button"].omrs-text-neutral:hover {
+  background-color: #eff1f2;
+}
+
+button.omrs-text-disabled,
+input[type="submit"].omrs-text-disabled,
+input[type="button"].omrs-text-disabled {
+  color: var(--omrs-ink-low-contrast);
+}
+
+/* Outlined Buttons */
+
+button.omrs-outlined-neutral,
+input[type="submit"].omrs-outlined-neutral,
+input[type="button"].omrs-outlined-neutral {
+  color: rgba(19, 19, 21, 0.6);
+}
+
+button.omrs-outlined-neutral:hover,
+input[type="submit"].omrs-outlined-neutral:hover,
+input[type="button"].omrs-outlined-neutral:hover {
+  background: #eff1f2;
+  color: rgba(19, 19, 21, 0.6);
+}
+
+button.omrs-outlined-action,
+input[type="submit"].omrs-outlined-action,
+input[type="button"].omrs-outlined-action {
+  color: var(--omrs-primary-blue);
+}
+
+button.omrs-outlined-action:hover,
+input[type="submit"].omrs-outlined-action:hover,
+input[type="button"].omrs-outlined-action:hover {
+  background: var(--omrs-interaction-minus-two);
+  color: var(--omrs-primary-blue);
+}
+
+button.omrs-outlined-destructive,
+input[type="submit"].omrs-outlined-destructive,
+input[type="button"].omrs-outlined-destructive {
+  color: var(--omrs-critical-alert-red);
+}
+
+button.omrs-outlined-destructive:hover,
+input[type="submit"].omrs-outlined-destructive:hover,
+input[type="button"].omrs-outlined-destructive:hover {
+  background-color: #f8eaea;
+  color: var(--omrs-critical-alert-red);
+}
+
+button.omrs-outlined-disabled,
+input[type="submit"].omrs-outlined-disabled,
+input[type="button"].omrs-outlined-disabled {
+  color: var(--omrs-ink-low-contrast);
+}
+
+/*setting border color for all outlined buttons*/
+
+button.omrs-outlined-destructive,
+input[type="submit"].omrs-outlined-destructive,
+input[type="button"].omrs-outlined-destructive,
+button.omrs-outlined-neutral,
+input[type="submit"].omrs-outlined-neutral,
+input[type="button"].omrs-outlined-neutral,
+button.omrs-outlined-action,
+input[type="submit"].omrs-outlined-action,
+input[type="button"].omrs-outlined-action,
+button.omrs-outlined-disabled,
+input[type="submit"].omrs-outlined-disabled,
+input[type="button"].omrs-outlined-disabled {
+  border: 1px solid var(--omrs-ink-low-contrast);
+}
+
+/* Rounded Button  */
+button.omrs-rounded,
+input[type="submit"].omrs-rounded,
+input[type="button"].omrs-rounded {
+  border-radius: 100px;
+}
+
+/* Button Sizes */
+
+button.omrs-rounded-lg,
+input[type="submit"].omrs-rounded-lg,
+input[type="button"].omrs-rounded-lg {
+  border-radius: 100px;
+  padding: 24px 130px;
+}
+
+button.omrs-btn-lg,
+input[type="submit"].omrs-btn-lg,
+input[type="button"].omrs-btn-lg {
+  border-radius: 8px;
+  padding: 24px 130px;
```

Comment:
same here about rem.

Rem is better for accessibility because people with vision problems set their font size larger and css that uses rem respects that whereas css using px does not.